core/src/Telemetry/iTelemetry.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Core
```

Comment:
Filename seems still to be with small i

Reply:
Seems still not updated. But this might be a tools issue more than anything.

---
In reply to: [208828746](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1173#discussion_r208828746) [](ancestors = 208828746)